extensions/wikia/PhalanxII/fallback/hooks/UserBlock.class.php

```diff
@@ -46,15 +46,15 @@ public static function blockCheck(User $user, &$block) {
 			return $cachedState['return'];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "return" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)